icons/solidjs/solidjs-plain-wordmark.svg

```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg">

```

Comment:
Please replace with the wordmark from here: https://www.solidjs.com/media

Then resize to fit inside 0 0 128 128, and lastly merge the paths and change the fill color to the primary color